JSDemos/Demos/Charts/Colorization/description.md

```diff
@@ -1 +1,14 @@
-The TreeMap component provides a&nbsp;flexible system of&nbsp;tile colorization. You can assign colors to&nbsp;tiles depending on&nbsp;their values or&nbsp;the group they belong&nbsp;to. If&nbsp;colors are specified in&nbsp;your data source, you can use them as&nbsp;well. Finally, the TreeMap includes nine built-in palettes that ensure the superb appearance.
\ No newline at end of file
+This demo shows techniques that you can use to colorize TreeMap tiles. To see each technique in action, use a drop-down editor below the component.
+
+To colorize the TreeMap tiles, you can use the [colorizer](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/) object. This object allows you to specify the colorization [type](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#type):
+
+- *"discrete"* (default)    
+This colorization algorithm paints each tile within a group in a color taken from the [palette](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#palette). When there are multiple groups, colorization begins from scratch in each group. Alternatively, you can force the UI component to continue colorization across groups. For this purpose, set the [colorizeGroups](//Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#colorizeGroups) property to `true`. It colorizes not each tile, but each group of tiles in a single color taken from the palette. This demo shows these two techniques in **Discrete** and **Grouped** modes.
```

Comment:
1. `When there are multiple groups, colorization begins from scratch in each group.`

What do you mean by "from scratch" in this context? Or do you mean:

"When there are multiple groups, colorization is applied to each group."

If not, please clarify.

2. `It colorizes not each tile, but each group of tiles in a single color taken from the palette.`

How is this different from the sentence in question #1 (above)? Please clarify.

Reply:
I'll try to explain it to you and maybe you will come up with some ideas how to make it more comprehensible:
 1) discrete mode **without** colorizeGroups applied
If there are mutiple groups, each group begins colorization for itself. It's like there is no continuity between the groups, they are not dependent on each other.
Imagine you have a palette of 4 colors: red, blue, green and yellow. And you have 3 groups, first has 3 items, second one has 2 items, third one has 2 items too. In this mode the colorization in this groups will look like that:
{ red item, blue item, green item }, { red item, blue item }, { red item, blue item }
That means that each group takes a palette and starts colorization once again, not looking at other groups.
2) discrete mode **with** colorizeGroups applied
That's a simpler one: each group takes a color from the palette and all the items in this group are colorized with this color.
Let's look at our previous example, 4 colors, 3 groups. The colorization would look like that:
{ red item, red item, red item }, { blue item, blue item }, { green item, green item }

Reply:
Based on your explanation, I came up with the following. Let me know if it's technically correct.

`- *"discrete"* (default)`  
`This colorization algorithm paints each tile within a group in a color from the [palette](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/colorizer/#palette). When there are multiple groups, the way in which groups are colorized depends on the colorizeGroups property setting:`

If colorizeGroups is set to `false`:
Each group is painted independently - regardless of the colors used in other groups.

If colorizeGroups is set to `true`:
Each group is painted with a different color from the palette.

Additionally, I noticed that in the demo, the Colorization Type can also be set to "Grouped".
Is this different from setting the colorizeGroups property to true/false as in your explanation above?
The reason I mention this is that since "Grouped" is an option type, as a reader I would expect to see a separate list item for *grouped* (similar to *discrete*, *gradient*, and *range*). Let me know what you think.

Reply:
Oh, I now understood what is so confucing about this demo. The thing is - **Grouped** "mode" in the demo is actually just a discrete mode with colorizeGroups set to `true`. This is a very old demo, and I can ask developers to change the naming, but I don't think it would be their top priority, so we gotta work with what we have. I think it's best to add something like this:

> If colorizeGroups is set to false (**Discrete** mode in the demo):
> Each group is painted independently - regardless of the colors used in other groups.

> If colorizeGroups is set to true (**Grouped** mode in the demo):
> Each group is painted with a different color from the palette.

Reply:
> If colorizeGroups is set to false (Discrete mode in the demo):
> Each group is painted independently - regardless of the colors used in other groups.

> If colorizeGroups is set to true (Grouped mode in the demo):
> Each group is painted with a different color from the palette.

Your suggestions sound fine.
Can you incorporate these into line 6 so I can see how the final text will look for this line?

Reply:
I updated the description.

Reply:
Looks good. Thank you!